### ISSUE TITLE
ignore URLs

### ISSUE THREAD

**Issue by Zamiell** [lasselupe33/eslint-plugin-comment-length#1]
Hello and thanks for the lint rule!

Unfortunately, it triggers many false positives with URLs, like the following:

```ts
/**
  * This function is based on the following code:
  * https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/member-ordering.md
  */
function foo() {}
```

Would it be possible to add a `ignoreUrls` option that defaults to `true`, like ESLint's [`max-len`](https://eslint.org/docs/rules/max-len) rule does?

Thank you!

**Comment by Zamiell** [comment]
For future readers, I decided to fork this rule and implement the functionality myself: https://github.com/IsaacScript/eslint-plugin-isaacscript/

**Comment by lasselupe33** [comment]
Thanks for the pull request (#2) @Zamiell which fixes this.
